mkdocs/nav.py

```diff
@@ -148,6 +186,63 @@ def __init__(self, title, url, path, url_context):
         self.next_page = None
         self.ancestors = []
 
+        self.markdown, self.meta = self.load_markdown()
+
+    def __eq__(self, other):
```

Comment:
`__eq__` or `__ne__` were added primarily to make testing easier, I am not sure if that is a good reason or not.